testsupport/wait/host.go

```diff
@@ -1588,7 +1614,7 @@ func UntilSpaceHasStatusTargetCluster(expected string) SpaceWaitCriterion {
 func (a *HostAwaitility) WaitUntilSpaceDeleted(name string) error {
 	a.T.Logf("waiting until Space '%s' in namespace '%s' is deleted", name, a.Namespace)
 	var s *toolchainv1alpha1.Space
-	err := wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
+	err := wait.Poll(a.RetryInterval, 2*a.Timeout, func() (done bool, err error) {
```

Comment:
Do we really need to double the timeout?
```suggestion
	err := wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
```

Reply:
I figured it was good to increase the default in this case because the default is 60 seconds and we have to account for 2 things:
1. spacecleanupcontroller won't delete a Space until it is at least 30 seconds old
2. Space won't be deleted until NSTemplateSet and all the namespace resources are deleted

That said, maybe 1 minute is still enough. I'll change it back to 1 minute and if it turns out to not be enough time then maybe we can add another 30 seconds to make it 90 seconds instead of 2 minutes.